Extension/src/LanguageServer/configurations.ts

```diff
@@ -386,6 +395,46 @@ export class CppProperties {
         }
     }
 
+    private nodeAddonMap: {[dependency: string]: string} = {
+      "nan": "node --no-warnings -e \"require('nan')\"",
+      "node-addon-api": "node --no-warnings -p \"require('node-addon-api').include\"",
+    };
+
+    private async addNodeAddonIncludeLocations(rootPath: string) : Promise<void> {
+        try {
+
+            const settings: CppSettings = new CppSettings(this.rootUri);
+            if (settings.addNodeAddonIncludePaths) {
+                const package_json = JSON.parse(await pfs.readFile(path.join(rootPath, "package.json"), "utf8"));
+                dependencyLoop:
+                for (const dep in this.nodeAddonMap) {
+                  if (dep in package_json.dependencies) {
+                      const execCmd: string = this.nodeAddonMap[dep];
+                      let stdout = await util.execChildProcess(execCmd, rootPath);
+                      if (!stdout) continue dependencyLoop;
```

Comment:
This can just be "continue", right? I don't see why the label is needed.

Reply:
i use labels so that if the code changes there is no need to worry about the scope of the continue. but i'm happy to change it.

Reply:
Yeah, I think the label should be removed. It is expected that in the rare a case an sub-loop is added inside another loop then any "continue" would need to be addressed.